packages/connectivity/android/src/main/java/io/flutter/plugins/connectivity/ConnectivityPlugin.java

```diff
@@ -158,6 +225,16 @@ private BroadcastReceiver createReceiver(final EventSink events) {
     return new BroadcastReceiver() {
       @Override
       public void onReceive(Context context, Intent intent) {
+        boolean isLost = intent.getBooleanExtra(ConnectivityManager.EXTRA_NO_CONNECTIVITY, false);
```

Comment:
It is not clear to me what does these code do. Could you explain? It doesn't seem to be related to the intention of this PR. I might be missing something though